bin/fuel-convert

```diff
@@ -10,6 +10,18 @@ from fuel.converters.base import MissingInputFiles
 from fuel.datasets import H5PYDataset
 
 
+class SplitOutputAction(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        absolute_path = os.path.abspath(values)
+        if os.path.isdir(absolute_path):
```

Comment:
@bartvm , I know, but I suggest that user can explicitly indicate that a path points to a directory by ending it with the separator `os.path.sep`. I am fine with two arguments though.